js/navigation.js

```diff
@@ -210,6 +211,11 @@ bcpl.navigation = (($, keyCodes) => {
 		}
 	};
 
+	const onBackButtonClicked = (clickEvent) => {
+		clickEvent.preventDefault();
```

Comment:
Couldn't you remove the click event from the button instead of doing this?

Reply:
@ascire i don't get this comment.

Reply:
I've been informed that `event.preventDefault()` is a sort of last-resort tool, since it blocks all other event handlers from propagating.  Is there a better way to get the intended effect?

Reply:
@ascire this is actually desired when someone click an element that triggers this method, they will be returned to the previous page. It can also be used if you put a click event on a anchor that you don't want to actually do what is supposed to...that is probably not great, but it happens.

Reply:
Fair enough.  I'll unblock.

Reply:
Also, it will only prevent the event handlers from firing on a button with the class `.window-back` which is desired.